lib/msal-common/src/client/UsernamePasswordClient.ts

```diff
@@ -102,6 +106,12 @@ export class UsernamePasswordClient extends BaseClient {
             parameterBuilder.addClaims(request.claims, this.config.authOptions.clientCapabilities);
         }
 
+        if (this.config.systemOptions.preventCorsPreflight) {
+            if (request.username) {
```

Comment:
Can we merge these two ifs into one?

Reply:
changed all occurrences to use `&&` instead of two `if`s